dbt_platform_helper/providers/cache.py

```diff
@@ -0,0 +1,76 @@
+import os
+from datetime import datetime
+from pathlib import Path
+
+import yaml
+
+
+class CacheProvider:
+    def __init__(self):
+        self.cache_file = ".platform-helper-config-cache.yml"
```

Comment:
Can we indicate privacy...
```suggestion
        self._cache_file = ".platform-helper-config-cache.yml"
```

I don't think the double underscore works on class variables, but you could do that by putting it in a `__method()` I guess.

Reply:
I think _ infront of the variable name is sufficient here - will commit it.